app/models/condition.rb

```diff
@@ -13,6 +13,7 @@ class Condition < ApplicationRecord
   before_update :create_validation_request!, if: -> { pre_commencement? && should_create_validation_request? }
 
   scope :sorted, -> { sort_by(&:sort_key) }
+  scope :cancelled, -> { where(cancelled_at: nil) }
```

Comment:
Maybe this should be called `not_cancelled` or something? Otherwise it might be confusing.